packages/core/src/sass/app/_common/layout/trading-hub-header.scss

```diff
@@ -90,7 +90,7 @@
 
     &__logo {
         height: 2.5rem;
-        margin: 0 0.5rem 0 1.5rem;
+        margin-bottom: 0.8rem !important;
```

Comment:
I have to use important to override one of our classnames which is used in the rest of the project and modifying that one may cause issues.